translations/ui-invoice/en.json

```diff
@@ -365,6 +365,7 @@
   "invoice.actions.approve.error.organizationIsNotExist": "Invoice cannot be approved because organization does not exist",
   "invoice.actions.approve.error.multipleFiscalYears": "Invoice cannot be approved because the funds used belong to multiple fiscal years",
   "invoice.actions.approve.error.outdatedFundIdInEncumbrance": "Invoice could not be approved. Encumbrance fund id does not match the fund id in the invoice line. Error must be corrected in database.",
+  "invoice.actions.approve.error.incorrectFundDistributionTotal": "Fund distribution total must add to 100% or totalPrice",
```

Comment:
sync with BE, it's not really what they need

Reply:
I have messaged Khamid and waiting for his response.